definitions/cleanup-workspace/cleanup-workspace.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cleanup-workspace
+spec:
+  params:
+    - name: subdirectory
+      type: string
+      description: The directory to remove within the workspace
+  workspaces:
+    - name: input
+      description: Workspace where the directory to cleanup exists
+  steps:
+    - name: cleanup
+      image: quay.io/hacbs-release/release-utils
+      script: |
+        #!/usr/bin/env sh
+        set -eux
+
+        CLEANUP_DIR="$(workspaces.input.path)/$(params.subdirectory)"
```

Comment:
Is it possible to pass an empty string as `subdirectory`? I have no idea tbh but, if so, we need a guard.

Reply:
Yes, it let me pass the empty string